extras/mini-stats/mini-stats.js

```diff
@@ -442,22 +88,74 @@ function MiniStats(app) {
 
     this.device = device;
     this.texture = texture;
-    this.wordAtlas = wordAtlas;
-    this.render2d = new Render2d(device);
+    this.wordAtlas = wordAtlasData.atlas;
+    this.render2d = new Render2d(device, options.colors);
     this.graphs = graphs;
     this.div = div;
 
     this.width = 0;
     this.height = 0;
-    this.gspacing = 2;
+    this.gspacing = 0.1;  // in percentage of hight, 10%
     this.clr = [1, 1, 1, 0.5];
 
     this._enabled = true;
 
-    this.resize(sizes[size].width, sizes[size].height, sizes[size].graphs);
+    // initial resize
+    this.activeSizeIndex = this._activeSizeIndex;
 }
 
+MiniStats.getDefaultOptions = function () {
+    return {
+
+        sizes: [
+            { width: 100, height: 16, graphs: false },
+            { width: 128, height: 32, graphs: true },
+            { width: 256, height: 64, graphs: true }
+        ],
+
+        startSizeIndex: 0,
+
+        colors: {
+            graph0: new Color(0.7, 0.2, 0.2, 1),
+            graph1: new Color(0.2, 0.7, 0.2, 1),
+            graph2: new Color(0.2, 0.2, 0.7, 1),
+            watermark: new Color(0.4, 0.4, 0.2, 1),
+            background: new Color(0, 0, 0, 0.4)
+        },
+
+        cpu: {
+            enabled: true,
+            watermark: 33
+        },
+
+        gpu: {
+            enabled: true,
+            watermark: 33
+        },
+
+        stats: [
+            {
+                name: "Frame",
+                stats: ["frame.ms"],
```

Comment:
It's unfortunate that this is now only available in profile and debug builds. Perhaps we can make an exception for his and #frame drawcalls and have them available in all builds? cc @willeastcott